runtime/cudaq/qis/qreg.h

```diff
@@ -25,7 +25,10 @@ concept ValidQregSize = N > 0;
 /// template parameters.
 template <std::size_t N = dyn, std::size_t Levels = 2>
   requires(details::ValidQregSize<N>)
-class qreg {
+class
+    // [[deprecated(
+    // "qreg deprecated and will be removed. use qvector or qarray.")]]
```

Comment:
We should probably have an action item to remove this deprecated feature.

Reply:
Will do